test/support/logger_support.rb

```diff
@@ -1,7 +1,7 @@
 module Runcible
   class Logger
     attr_accessor :message
-    alias_method :debug, :message=
-    alias_method :error, :message=
+    alias debug message=
+    alias error message=
```

Comment:
What is the difference between the two? If alias is more widely used now then let's go with that.